packages/tinyemu/tinyemu_patch.patch

```diff
@@ -0,0 +1,122 @@
+diff -uNr tinyemu-2019-12-21/Makefile tinyemu-2019-12-21.mod/Makefile
+--- tinyemu-2019-12-21/Makefile	2019-12-22 00:23:54.000000000 +0700
++++ tinyemu-2019-12-21.mod/Makefile	2022-04-03 14:47:01.090508344 +0700
+@@ -50,7 +50,7 @@
+ CFLAGS+=-D_GNU_SOURCE -DCONFIG_VERSION=\"$(shell cat VERSION)\"
+ LDFLAGS=
+ 
+-bindir=/usr/local/bin
++bindir=@TERMUX_PREFIX@/local/bin
+ INSTALL=install
+ 
+ PROGS+= temu$(EXE)
+diff -uNr tinyemu-2019-12-21/fs_net.c tinyemu-2019-12-21.mod/fs_net.c
+--- tinyemu-2019-12-21/fs_net.c	2019-12-22 00:23:54.000000000 +0700
++++ tinyemu-2019-12-21.mod/fs_net.c	2022-04-03 14:47:11.670508345 +0700
+@@ -2093,7 +2093,7 @@
+ static FSDevice *fs_import_fs;
+ #endif
+ 
+-#define DEFAULT_IMPORT_FILE_PATH "/tmp"
++#define DEFAULT_IMPORT_FILE_PATH "@TERMUX_PREFIX@/tmp"
+ 
+ FSDevice *fs_net_init(const char *url, void (*start_cb)(void *opaque),
+                       void *start_opaque)
+diff -uNr tinyemu-2019-12-21/readme.txt tinyemu-2019-12-21.mod/readme.txt
+--- tinyemu-2019-12-21/readme.txt	2019-12-22 00:23:54.000000000 +0700
++++ tinyemu-2019-12-21.mod/readme.txt	2022-04-03 14:47:11.642508345 +0700
+@@ -44,7 +44,7 @@
+ 
+ - Use 'make' to compile the binaries.
+ 
+-- You can optionally install the program to '/usr/local/bin' with:
++- You can optionally install the program to '@TERMUX_PREFIX@/local/bin' with:
+ 
+   make install
+ 
+@@ -74,14 +74,14 @@
+ 
+   ./temu rv128test/rv128test.cfg
+ 
+-- Access to your local hard disk (/tmp directory) in the guest:
++- Access to your local hard disk (@TERMUX_PREFIX@/tmp directory) in the guest:
+ 
```

Comment:
You don't need to fix hardcoded paths in `readme.txt` which is not installed (I suppose).